source/security.rst

```diff
@@ -23,8 +23,19 @@ Limitations
 
 - **HTTP Traffic to Origin Servers**: Traffic to backend services, including computational resources like Jupyter servers, is currently over HTTP, which is unencrypted. Plans are underway to upgrade this to HTTPS to ensure encryption of data in transit, thereby bolstering security.
 
-Security Controls
------------------
+Controls
+^^^^^^^^
+
+These are things the the out of the box OnDemand installation will provide
+that some centers may want to change or disable altogether.
+
+- **File Access**: OnDemand lets users navigate the file system. While file permissions
+  limit what a user can view and navigate to, some centers may want to limit this even further.
+  One option is to :ref:`set-file-allowlist` to limit what directories users may navigate to.
```

Comment:
This addition was really what I was getting at in comments in the previous PR.

Instead of just blind links to _all_ customizations - we have specific callouts to specific behaviors that an initial installation may want to change. With a link to that specific configuration.

Reply:
Thanks for explaining, that looks good to me.